modules/check_in/app/services/v2/lorota/client.rb

```diff
@@ -78,11 +78,7 @@ def data(token:)
       def connection
         Faraday.new(url: url) do |conn|
           conn.use :breakers
-          if Flipper.enabled?('check_in_experience_lorota_401_mapping_enabled')
-            conn.response :raise_error, error_prefix: 'LOROTA-MAPPED-API'
-          else
-            conn.response :raise_error, error_prefix: service_name
-          end
+          conn.response :raise_error, error_prefix: 'LOROTA-MAPPED-API'
```

Comment:
We should change the prefix to refer to `service_name`

```suggestion
          conn.response :raise_error, error_prefix: service_name
```

Reply:
Replacing prefix to `service_name` would miss `401` exception configured in [here](https://github.com/department-of-veterans-affairs/vets-api/blob/e730247344850c5ab36a051e33a56d8344544922/modules/check_in/config/locales/exceptions.en.yml#L23-L28)

Reply:
Sure @randomsync, can be updated as it is minor change.  Thought `LOROTA-MAPPED-API` the name decided at the time of implementation.

Reply:
It was updated to be able to use the feature flag. Since we're removing the flag, it should be changed back to `LOROTA-API...`